test/JDBC/expected/BABEL_1940.out

```diff
@@ -0,0 +1,242 @@
+
```

Comment:
Basic test cases mentioned in the Jira is missing.

Reply:
Added one, everything else is covered